lsp4helidon/jdt-extension/jdt-extension-core/src/main/java/io/helidon/core/internal/providers/HelidonPropertiesProvider.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.core.internal.providers;
```

Comment:
all the packages in this repo start with `io.helidon.build`.
Here it should be something like this `io.helidon.build.lsp.jdt.core`